### PR TITLE
add supplement in the doc

### PR DESCRIPTION
**Pull request by flobb**
- Specify to edit the override `docker.service` instead of the original one that must not be edited
- add a bit of explaination

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,8 +136,14 @@ And listen to interfaces `lo` and `docker0`.
...
 
     # For systemd users (Fedora and recent Ubuntu versions) :
-    $ vim /lib/systemd/system/docker.service
-    # append the --bip="172.17.42.1/24" --dns="172.17.42.1" options to the ExecStart line
+    
+    $ vim /etc/systemd/system/docker.service
+    
+    # append
+    [Service]
+    ExecStart=
```
Comment (jderusse) [inline README.md:144]: I'm not sure the line `ExceStart` should be dupplicated

File: README.md [hunk 2]
```diff
@@ -133,8 +136,14 @@ And listen to interfaces `lo` and `docker0`.
...
+    ExecStart=/usr/bin/docker daemon -H fd:// --bip=172.17.42.1/16
```
Comment (jderusse) [inline README.md]: Why do you remove the option `--dns`?

File: README.md (hunk 1 above)
Comment (flobb) [inline README.md:144]: I don't know why, but it don't work if you don't do it (and it's specified like that in the docker doc)

File: README.md (hunk 2 above)
Comment (flobb) [inline README.md]: Because i don't know how to do a C/C ;)

**Comment by flobb** [comment]
Fixed

**Comment by jderusse** [comment]
Thank you

**Pull request merged**